httpx/client.py

```diff
@@ -621,9 +621,9 @@ def redirect_stream(
 
         return response
 
-    def dispatcher_for_url(self, url: URL) -> Dispatcher:
+    def AsyncDispatcher_for_url(self, url: URL) -> AsyncDispatcher:
```

Comment:
This was probably not intended, was it? :)

Reply:
oh man.